src/test/java/org/cactoos/iterable/RangeOfTest.java

```diff
@@ -72,35 +72,35 @@ public Integer apply(
                 )
             ),
             Matchers.contains(1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89)
-        );
+        ).affirm();
     }
 
     @Test
     public final void testLongRange() {
-        MatcherAssert.assertThat(
-            "Can't generate a range of long",
+        new Assertion<>(
+            "Must generate a range of long",
             new ListOf<>(
                 new RangeOf<>(1L, 5L, value -> ++value)
             ),
             Matchers.contains(1L, 2L, 3L, 4L, 5L)
```

Comment:
@baudoliver7 Here it can be replaced with `HasValues`

Reply:
@andreoss Yes, you are right, but I noticed that the scope of this ticket is to replace `MatcherAssert.assertThat` by `Assertion`.